.github/workflows/ci.yml

```
@@ -57,7 +57,7 @@ on:
 
 jobs:
   fmt:
-    name: cargo fmt
+    name: check formatting
```

I think I'd slightly prefer a more "pithy" name - this is already a GitHub "check".

```suggestion
    name: formatting
```
I do agree that this has outgrown the `cargo fmt` name.